resources-generator/src/main/kotlin/dev/icerock/gradle/utils/ProcessResourcesUtils.kt

```diff
@@ -22,8 +22,8 @@ fun dependsOnProcessResources(
         .configureEach { processResourcesTask ->
             processResourcesTask.exclude {
                 val path: String = it.file.absolutePath
-                if (shouldExcludeGenerated && path.contains("generated/moko")) return@exclude true
-                if (path.contains("resources/MR")) return@exclude true
+                if (shouldExcludeGenerated && path.contains("generated/moko-resources")) return@exclude true
```

Comment:
i think this logic (all function `dependsOnProcessResources`) not needed anymore, can we check?

Reply:
removed from apple target, reworked for jvm, js